### PR TITLE
GenStage 0.11.0 release

### PR DESCRIPTION
**Pull request by amokan**
I started a dummy project this morning to evaluate honeydew and noticed there was also a GenStage update today that removed the `Experimental` namespace (see [CHANGELOG.md#v0110](https://github.com/elixir-lang/gen_stage/blob/master/CHANGELOG.md#v0110)) which prevented me from from getting it running.

**Changes Made**
- removed all `Experimental` aliases for GenStage from modules
- removed all full namespace references in tests with `Experimental` prefixes for GenStage
- fixed URLs to GenStage documentation in README

**Not Done**
- `test/honeydew/queue/erlang_queue_integration_test.exs:41` is failing, but does not appear to be related to these changes. I will pull another version and pin GenStage to v0.10.0 to verify
- I did not mess with your version number or anything in the mix.exs file for Honeydew. Figured I'd let you handle that if you accept this pull :)

### CODE REVIEW

**Comment by amokan** [comment]
Looks like the failing test I ran into is related to the async stuff and others appear to fail sometimes due to, what I assume, is a task finishing before you have a chance to cancel and so-on. My branch will pass on that test randomly also. I'm sure you've dealt with that before?

TLDR; I ran the test suite with your master branch pinned to GenStage v0.10.0 as well as my version on v0.11.0 - both behave similar.

**Comment by koudelka** [comment]
Hey @amokan, thanks for the PR, looks great. :)

I've fixed the test issue (and others) that you saw, let me know if anything else crops out.

Drop me a line if you feel like there's anything missing from Honeydew during your evaluation.

**Comment by amokan** [comment]
Will do @koudelka 

One thing - (_hopefully I didn't cause it_) but your `doc` directory is missing in the repo it appears so your nice diagrams on the README are missing.

**Comment by koudelka** [comment]
Damn, oops, thanks. 

Looks like `ex_doc` overwrote the directory without a warning and I committed the changes. :\
